packages/bsky/src/services/indexing/processor.ts

```diff
@@ -246,6 +247,30 @@ export class RecordProcessor<T, S> {
     }
   }
 
+  async sendNotifsToUser(notifs: Notif[]) {
```

Comment:
heads up, we'll probably want to do the actual _sending_ of notifications from within the indexing service (not the record processor). the `insert/update/deleteRecord` functions on the processor would probably want to return some information about notifications that the indexing service could then send off through the gorush client. We'll want to wire the gorush client up through the `AppContext` and give the indexing service access to it

Reply:
Done! Thanks for the heads up